### ISSUE TITLE
SPARQL optional failing

### ISSUE THREAD

**Issue by mathib** [comunica/comunica#666]
#### Issue type:

- :question: Question

____
#### Question:

Not sure if this is a bug or me just missing things. When I launch a query as follows:
```
SELECT DISTINCT ?s ?opt
   ?s <http://myprop/prop> ?o .
   OPTIONAL { ?s <http://myprop/otherprop> ?opt }
}
```
I get correct results if at least one result contains a value for ?opt. However, when none of my RDF datasets match that optional pattern, I get no results and receive the following error:

`comunica-browser.js:136386 Uncaught (in promise) Error: Variable '?opt' is used in the projection result, but is not assigned.`

If this is not a Comunica bug, are there any pointers how to deal with this situation (variable in OPTIONAL not resolving)?

Using Comunica CDN (latest)

**Comment by rubensworks** [comment]
That definitely is a bug, thanks for reporting!

**Comment by rubensworks** [comment]
This should be fixed in 1.13.1, could you confirm on your end?

**Comment by mathib** [comment]
did a quick test on my side and it seems to be solved! Thanks
